### ISSUE TITLE
Cutting TLS_RSA_WITH_3DES_EDE_CBC_SHA from our ciphersuites

### ISSUE THREAD

**Issue by konklone** [18F/cg-provision#244]
We specifically added 3DES to our list of preferred ciphersuites when we were first configuring ELBs in 2014 and 2015. This was generally to support IE8, so that even if our websites didn't well-support IE8, it wouldn't just drop the connection entirely with no user ecourse.

However, times have changed -- we're already cutting off IE8 on XP by requiring SNI at the CloudFront layer for our static sites, and in 2017, IE8 use has diminished to negligible levels. 

You can see 3DES in use in 18f.gsa.gov here:
https://www.ssllabs.com/ssltest/analyze.html?d=18f.gsa.gov&s=52.84.0.10&latest

`TLS_RSA_WITH_3DES_EDE_CBC_SHA` is getting rightly flagged as weak, at 112 bits. It's time we cut this cipher loose across our infrastructure.

**Comment by konklone** [comment]
Sorry, to use an example I hope is more relevant to this repo, the cipher is also in use on dashboard.fr.cloud.gov:

https://www.ssllabs.com/ssltest/analyze.html?d=dashboard.fr.cloud.gov&s=52.222.66.41&latest

18f.gsa.gov is terminated via a CloudFront distro via Federalist.

**Comment by sharms** [comment]
Hi Eric - we conform to the most recent Amazon provided standards for their services:

All hosted apps: http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-policy-table.html

CloudFront: http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/secure-connections-supported-viewer-protocols-ciphers.html

**Comment by brittag** [comment]
To assist with status tracking, [here is the associated Favro card for this topic](https://favro.com/card/1e11108a2da81e3bd7153a7a/18F-5159).

**Comment by konklone** [comment]
As an update, a new federal-wide binding directive from DHS bans 3DES (and RC4) from use: https://cyber.dhs.gov

With a deadline of 120 days from 10/16.

**Comment by kynetiv** [comment]
Just chiming in that the College Scorecard team received the above notice to `Ensure 3DES and RC4 ciphers are disabled on web servers.` The deadline appears to be Feb. 20, 2018.

**Comment by brittag** [comment]
@kynetiv Yes, thanks! AWS released the feature update that allows us to make this change, and we plan on it.
